qemu.mk

```diff
@@ -179,9 +181,8 @@ ifneq ($(filter check,$(MAKECMDGOALS)),)
 CHECK_DEPS := all
 endif
 
-check-args := --bios $(ROOT)/out/bios-qemu/bios.bin
```

Comment:
Shouldn't that be `bl1.bin` like in this patch: https://github.com/OP-TEE/build/commit/46ddff2b339b9457a37b3bd58b29e4a4e06dc10f#diff-e0bd85e5484a3cb990cdad4e8a7bcea8L129?

Reply:
With this line replaced by: `check-args := --bios $(BINARIES_PATH)/bl1.bin` (and the `+=` kept below):
`Tested-by: Jerome Forissier <jerome.forissier@linaro.org>`

(have you actually tried `make check`?)

Reply:
Yes I tried `make check` and failed to get something out of it. I was even wondering whether we maintain this part...  Note: see https://github.com/OP-TEE/build/pull/254.

Reply:
That said, should we really set `check-args := --bios $(BINARIES_PATH)/bl1.bin`?
Note: in the qemu_v8.mk, the bios is not 'checked'.